charts/firefly-signer/templates/keystore.yaml

```diff
@@ -0,0 +1,41 @@
+{{/*
+  Copyright © 2024 Kaleido, Inc.
+
+  SPDX-License-Identifier: Apache-2.0
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+  http://swww.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/}}
+
+{{- if .Values.signer.enabled }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "firefly-signer.fullname" . }}-keystore
+  labels:
+    {{- include "firefly-signer.labels" . | nindent 4 }}
+stringData:
+  75a99473917701038e854ef6999c76cd947c9f9e: |-
+    {"address":"75a99473917701038e854ef6999c76cd947c9f9e","id":"8cfd0d87-b5ac-45a4-acb9-55f5b2b9bb2e","version":3,"crypto":{"cipher":"aes-128-ctr","ciphertext":"72c8d1f61787f7b1455d7e081edd72d7e1f41f503f363cb7afe08634c10bcd09","cipherparams":{"iv":"8731e724016127b91cf013bc0b4f7f82"},"kdf":"scrypt","mac":"af4de886e6a7e22beefbeaea213e621f00ebca859528c7d474635850800a8de6","kdfparams":{"dklen":32,"n":1024,"p":1,"r":8,"salt":"b1e3901cfe16ed73c538e1adaef5c584ea3b0a405ec1dd1c8776e55c3385c27b"}}}
+  75a99473917701038e854ef6999c76cd947c9f9e.toml: |-
+    [metadata]
+    createdAt = 2019-11-05T08:15:30-05:00
+    description = "File based configuration"
+
+    [signing]
+    type = "file-based-signer"
+    key-file = "/data/keystore/75a99473917701038e854ef6999c76cd947c9f9e"
+    password-file = "/data/password"
+  password: correcthorsebatterystaple
```

Comment:
Is there a script to be able to generate a key and add it? Would be nice to have, maybe a follow up issue

Reply:
or even a way to import keys!

Reply:
See https://github.com/hyperledger/firefly-helm-charts/pull/76#issuecomment-2186095649